skllm/llm/gpt/mixin.py

```diff
@@ -77,12 +83,12 @@ def _get_openai_key(self) -> str:
         -------
         openai_key: str
         """
-        key = self.openai_key
-        if key is None:
-            key = _Config.get_openai_key()
-        if key is None:
+        openai_key = self.key
+        if openai_key is None:
+            openai_key = _Config.get_openai_key()
+        if openai_key is None:
             raise RuntimeError("OpenAI key was not found")
-        return key
+        return openai_key
 
     def _get_openai_org(self) -> str:
```

Comment:
Since we already touched the refactoring of this mixin, it would be nice to also remove all the references of openai as these names are very unfortunate remnants from the early versions.  

P.S. If you don't want to spend time on this, it is also fine to leave it as is.